### PR TITLE
Status/Error handler routing and httpErrors configuration in web config #1096 

### LINKED ISSUE #1058: The link to 'Humanitarian Toolbox' in the page footer should open a new window/tab
The page footer includes "Powered by Humanitarian Toolbox". Suggest making the link open a new window/tab. Picky, I know, but if you're taking a user to a different website, they won't lose their place on allReady. 

Version: 1.0.0.0 ddb2198

### LINKED ISSUE #1073: App Needs Status Code Pages
The app needs [status code pages](https://docs.asp.net/en/latest/fundamentals/error-handling.html) (most notably a 404 page).

### LINKED ISSUE #1092: On the Admin Organization list page, make the organization website URLs clickable
Open the link in a new tab/window, since you're taking the user away from allReady:
adminorglisturl

Discovered in Version: 1.0.0.0 8a77b3a

### PR DESCRIPTION
**Pull request by binaryjanitor**
Status/Error handler routing and httpErrors configuration in web config #1096 

Fixes #1073 
Fixes #1092 
Fixes #1058

### CODE REVIEW

**Comment by tonysurma** [comment]
@mheggeseth can you do a quick review of this?

File: AllReadyApp/Web-App/AllReady/web.config [hunk 1]
```diff
@@ -4,6 +4,10 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath=".\AllReady.exe" arguments="" forwardWindowsAuthToken="false" stdoutLogEnabled="false" />
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" forwardWindowsAuthToken="false" stdoutLogEnabled="false" />
```
Comment (mheggeseth) [inline AllReadyApp/Web-App/AllReady/web.config]: Please remove this change until we get our act together w.r.t. deploying on Azure with environment variables.

**Comment by mheggeseth** [comment]
`404.cshtml` will be presented inside `_Layout.cshtml` so the full HTML document declaration isn't needed; only the markup you want to put in the body. In other words, you can delete everything except the innards of your `<body>` tag.

As for what to do with your styles, you can add a `@RenderSection("styles", required: false)` to the `<head>` tag of `_Layout.cshtml` and then wrap your `<style>` tag in 404 with `@section styles { ... }`

**Comment by mheggeseth** [comment]
Oh, and give the view a title:
```
@{
  ViewBag.Title = "I still haven't found what I'm looking for"; //or whatever
}
```

**Comment by binaryjanitor** [comment]
@mheggeseth done

**Comment by mheggeseth** [comment]
:shipit:

**Pull request merged**